src/datastructures.jl

```diff
@@ -291,13 +399,44 @@ Returns the `colors` field of a `Connection` `conn`.
 """
 get_colors(conn::Connection) = conn.colors
 
+"""
+    get_inv_data(design::Connection)
+
+Returns the `inv_data` field of a `Connection` `design`.
+"""
+get_inv_data(design::Connection) = design.inv_data
+
 """
     get_plots(conn::Connection)
 
 Returns the `plots` field of a `Connection` `conn`.
 """
 get_plots(conn::Connection) = conn.plots
 
+"""
+    get_inv_times(data)
```

Comment:
```suggestion
    get_inv_times(data)
    get_inv_times(design::AbstractGUIobj) 
```

Generally suggestion for all subsequent types as well. This allows a user to directly see all methods.